.github/workflows/rocm_wheels.yml

```diff
@@ -0,0 +1,66 @@
+name: rocm-wheels
+
+on:
+  push:
+    branches:
+      - develop
+  workflow_dispatch:
+
+jobs:
+  target_determinator:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+    - id: set-matrix
+      shell: python
+      run: |
+        import os
+        import json
+        environ = os.environ
+
+        PY_VERSIONS = ['3.11']
+
+        include = []
+        for os in ['ubuntu-alola']:
+          for python in PY_VERSIONS:
+            for torch_version in ['2.4.0']:
+              for toolkit_type, toolkit_short_versions in {'rocm': ["6.0", "6.1"]}.items():
+                for toolkit_short_version in toolkit_short_versions:
+                  include.append(dict(
+                    os=os,
+                    python=python,
+                    torch_version=torch_version,
+                    toolkit_type=toolkit_type,
+                    toolkit_short_version=toolkit_short_version,
+                  ))
+                  print(include[-1])
+        matrix = {'include': include}
+        print(json.dumps(matrix))
+        with open(environ["GITHUB_OUTPUT"], "a") as fd:
+          fd.write("matrix="+json.dumps(matrix))
```

Comment:
I don't think we need a target determinator to create the matrix, because it's static.
So I think you can put the matrix [directly](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#using-a-matrix-strategy)